app/mailers/user_update_mailer.rb

```diff
@@ -8,6 +8,7 @@ def new_profile_email(person, by_email = nil)
     @person = person
     @by_email = by_email
     @profile_url = profile_url(person)
+    @firefox_browser_warning = t('.firefox_message', default: '')
```

Comment:
you should be able to DRY these out using
```
before_action :set_firefox_warning,
    :new_profile_email,
    :updated_profile_email,
    :deleted_profile_email
```

and then at bottom of page in private section

```
private

  def set_firefox_warning
    @firefox_browser_warning = t('.firefox_message', default: '')
  end
```